examples/data_loss_prevention/dlp_stages/gliner_triton.py

```diff
@@ -112,9 +114,9 @@ def _load_label_data(self):
             label_data = torch.load(self._labels_file)
 
             labels_embeddings = label_data['embeddings']
-            if sorted(self._labels) != sorted(label_data['labels']):
-                raise ValueError("Label mismatch between model and label file.")
+            self._labels = label_data['labels']
         else:
+            self._labels = self._user_labels
```

Comment:
(Just line 119) Do we want to have any sort of validation here? Just permitting an arbitrary list[str] may not be desirable. Or does calling `model.encode_labels` perform the validation?

Reply:
There isn't much validation that this stage is able to perform. The only real requirement for the labels is that they match the keys of the regex patterns that is the type of pii we are searching for ('ssn', 'credit_card'...)